mmyolo/datasets/transforms/mix_img_transforms.py

```diff
@@ -38,15 +38,7 @@ def __init__(self,
 
         self.max_refetch = max_refetch
         self.prob = prob
-
-        if pre_transform is None:
-            self.pre_transform = None
-        else:
-            _transforms = []
-            for transform in pre_transform:
-                if isinstance(transform, dict):
-                    _transforms.append(TRANSFORMS.build(transform))
-            self.pre_transform = Compose(_transforms)
+        self.pre_transform = Compose(pre_transform)
```

Comment:
Thank you very much for your contribution.

It's better

```python
 if pre_transform is None:
       self.pre_transform = None
 else:
       self.pre_transform = Compose(pre_transform)
```
This avoids some unnecessary calculations of the L115